plugin/forward/health_test.go

```diff
@@ -83,6 +114,47 @@ func TestHealthTimeout(t *testing.T) {
 	}
 }
 
+func TestHealthTimeoutNoRecursion(t *testing.T) {
```

Comment:
this is duplicating other tests

Reply:
Redid both tests i added. Now checks the RD-flag in the dns-requests.